### PR TITLE
helm: support using image digest instead of tag

### PR DESCRIPTION
**Pull request by consideRatio**
A change as suggested by @kvaps in https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/issues/19#issuecomment-734427533.

### CODE REVIEW

File: deploy/helm/values.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ replicaCount: 1
 image:
   repository: quay.io/kubernetes_incubator/nfs-provisioner
   tag: v2.3.0
+  digest: ""
```
Comment (kvaps) [inline deploy/helm/values.yaml]: Thanks for your PR!
Could you please bump the chart version and add `digest` into the [configuration reference](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/tree/master/deploy/helm#configuration)

Comment (consideRatio) [reply deploy/helm/values.yaml]: Done, I bumped the minor version as it was a new functionality for the chart even though it was very minor.

**Comment by kvaps** [comment]
Thanks, digest now available in helm chart version 1.2.0

**Pull request merged**